nano/README.md

```diff
@@ -8,6 +8,35 @@ template engine.
 This README is not yet complete, but gives a basic rundown of how the folder structure
 works. I'll get around to finishing it one day.
 
+## Contributors
+
+This section is for listing the contributors to NanoUI.
+
+ - -tg-station/Neersighted
+  - Special thanks goes to Neersighted for creating most of the newest and best features
+    of NanoUI. Everything between using the Gulp build system to switching to LESS styles
+    has been directly adapated and/or copied from his original work for -tg-station and
+    their UI system, tgui.
+
+    Systems ported/adapted from -tg-station and Neersighted include, but are not limited
+    to:
+
+    - NanoUI's JavaScript controlling the "fancy" borderless window mode.
```

Comment:
A little more detailed than needed but sure, I don't mind.
I would also add baystation to this list for completeness, especially Faerdan and PsiOmegaDelta.